examples/algorithms/mst/mst.cu

```diff
@@ -0,0 +1,109 @@
+#include <gunrock/algorithms/algorithms.hxx>
+#include <gunrock/algorithms/mst.hxx>
+#include "mst_cpu.hxx"  // Reference implementation
+#include <cxxopts.hpp>
+
+using namespace gunrock;
+using namespace memory;
+
+struct parameters_t {
+  std::string filename;
+  cxxopts::Options options;
+
+  /**
+   * @brief Construct a new parameters object and parse command line arguments.
+   *
+   * @param argc Number of command line arguments.
+   * @param argv Command line arguments.
+   */
+  parameters_t(int argc, char** argv)
+      : options(argv[0], "Sparse Matrix-Vector Multiplication example") {
+    // Add command line options
+    options.add_options()("help", "Print help")                      // help
+        ("m,market", "Matrix file", cxxopts::value<std::string>());  // mtx
+
+    // Parse command line arguments
+    auto result = options.parse(argc, argv);
+
+    if (result.count("help") || (result.count("market") == 0)) {
+      std::cout << options.help({""}) << std::endl;
+      std::exit(0);
+    }
+
+    if (result.count("market") == 1) {
+      filename = result["market"].as<std::string>();
+      if (util::is_market(filename)) {
+      } else {
+        std::cout << options.help({""}) << std::endl;
+        std::exit(0);
+      }
+    } else {
+      std::cout << options.help({""}) << std::endl;
+      std::exit(0);
+    }
+  }
+};
```

Comment:
This is great! An option to avoid CPU reference code will be nice when you just want to test for performance.
- `--validate` for example.